ComfyUI/tests-unit/comfy_api_nodes_test/test_luma_text2img.py

```
@@ -0,0 +1,225 @@
+"""
+Tests for Luma Text2Img node functionality.
+
+This module tests the LumaImageGenerationNode which provides text-to-image generation
+capabilities using the Luma API.
+"""
+
+import pytest
+import torch
+from unittest.mock import Mock, patch, MagicMock
```

Ruff (0.12.2)

`unittest.mock.MagicMock` imported but unused

Remove unused import: `unittest.mock.MagicMock`

(F401)